### PR TITLE
Fix two HTTP tests

### PR DESCRIPTION
**Pull request by eriktews**
The new version of Firefox returns the HTTP POST body as key-value pairs and does not provide raw access to the body. This patch adjusts the tests to expect this behaviour.

### CODE REVIEW

File: test/test_http_instrumentation.py [hunk 1]
```diff
@@ -687,14 +687,18 @@ def test_record_post_data_ajax_no_key_value(self):
...
+            "test@example.com   name surname": [None]}
 
     def test_record_post_data_ajax_no_key_value_base64_encoded(self):
         """Test Base64 encoded AJAX payloads (no key=value form)."""
         post_format = "noKeyValueBase64"
         db = self.visit("/post_request_ajax.html?format=" + post_format)
         post_body = self.get_post_request_body_from_db(db)
-        assert post_body == "dGVzdEBleGFtcGxlLmNvbSArIG5hbWUgc3VybmFtZQ=="
+        assert json.loads(post_body) == {
+            "dGVzdEBleGFtcGxlLmNvbSArIG5hbWUgc3VybmFtZQ": [""]}
```
Comment (nhnt11) [inline test/test_http_instrumentation.py]: Could you please file an issue to investigate this in Firefox? Assuming that we are right that Firefox is the culprit here, this is definitely a bug that the folks who work on WebExtension APIs will want to know about and fix.

Comment (nhnt11) [reply test/test_http_instrumentation.py]: This particular case is especially eye-catching since the key-value parsing is stripping the "==" base64 padding.
